src/masonite/drivers/notification/DatabaseDriver.py

```diff
@@ -0,0 +1,41 @@
+"""Database driver Class."""
+import json
+from masonite.app import App
+from masonite import Queue
+
+from ..models import DatabaseNotification
```

Comment:
Not sure of the issues this will cause but in other database drivers I just get the application and call `self.application.make('builder')`. Maybe we should do that? This calls the query builder that is bound in the kernel class

Reply:
Finally I am not sure of the impact of what you're suggesting,
Here I am importing the model, do you want that we don't import the model and use only table name like this :
```
self.application.make('builder').table("notifications").create(model_data)
# instead of 
from ..models import DatabaseNotification
DatabaseNotification.create(model_data)
``` 

I want to be as flexible as possible. This feature is needing a table ("notifications") and the related model Notification provided in the codebase. Maybe we should give the user ability to define its own table name ?

Reply:
oh no okay I have seen what was done in QueueDatabaseDriver, really nice. I have done the same now